aidies-project/client/src/components/CampaignBrowser.js

```diff
@@ -22,23 +21,27 @@ const useStyles = makeStyles((theme) => ({
 const CampaignBrowser = () => {
     const classes = useStyles();
     const campaigns = useSelector(state => state.campaign.list );
+
     const dispatch = useDispatch();
     useEffect(() => {
         dispatch(getCampaigns())
     }, []);
     
     if(!campaigns.length){
-        return null;
+        return null; 
     }
     return (
-        <div className={classes.root} className='campaign-browser'>
+    
+        <div className={`${classes.root} campaign-browser`} >
+            
```

Comment:
Minor spacing suggestion!
```suggestion
        <div className={`${classes.root} campaign-browser`} >
```